doc/man/8/ceph-disk.rst

```diff
@@ -176,6 +193,62 @@ Usage::
 
 Here, [PATH] is path to a block device or a directory.
 
+deactivate
+----------
+Deactivate the Ceph OSD. It stops osd daemo and try to mark this osd out
+(optional). To prevent OSD activate by ``udev``, remove the *ready*, *active*,
+*INIT-specific* files and add the *deactive* file. Last, unmount the currect
+osd location. Futhermore, if the osd is dmcrypt, remove the data dmcrypt map. Atfer
+that, osd will remain the ``down`` status, you can activate with
+:option:`--reactivate`. Now, It is only run directly. It can be triggered
+by :program:`ceph-deploy` in later version.
+
+Usage::
+
+	ceph-disk deactivate [PATH]
+
+Here, [PATH] is path to a block device or a directory.
+
+Another option :option:`--mark-out` can also be used with this subcommand.
+``--mark-out`` marks the OSD out. It will trigger the object remapped. If
```

Comment:
s/It will trigger the object remapped./The objects it contains will be remapped/